lib/scrabble.rb

```diff
@@ -15,4 +14,54 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  def score(word)
+    score = 0
+    if word != nil
+      word_array = word.chars
+      word_array.each do | letter |
```

Comment:
When you are iterating over a collection and modifying an outside variable, there is a better enumerable to use